### PR TITLE
Adds skin presets for Vaurcae, based on brood colors from the wiki

### PR DESCRIPTION
**Pull request by feartheblackout**
title

### CODE REVIEW

**Comment by feartheblackout** [comment]
!wip

**Comment by feartheblackout** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/modules/client/preference_setup/general/03_body.dm [hunk 1]
```diff
@@ -618,17 +616,20 @@ var/global/list/valid_bloodtypes = list(
 			return TOPIC_REFRESH_UPDATE_PREVIEW
 
 	else if(href_list["skin_color"])
-		if(!has_flag(mob_species, HAS_SKIN_COLOR) && !has_flag(mob_species, HAS_SKIN_PRESET))
+		if(!has_flag(mob_species, HAS_SKIN_COLOR))
 			return TOPIC_NOACTION
-		if(has_flag(mob_species, HAS_SKIN_COLOR) && !has_flag(mob_species, HAS_SKIN_PRESET))
+		if(has_flag(mob_species, HAS_SKIN_COLOR))
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/general/03_body.dm:621]: this is a useless check now, it's the same as the one above

File: code/modules/client/preference_setup/general/03_body.dm [hunk 2]
```diff
@@ -618,17 +616,20 @@ var/global/list/valid_bloodtypes = list(
...
 				pref.r_skin = GetRedPart(new_skin)
 				pref.g_skin = GetGreenPart(new_skin)
 				pref.b_skin = GetBluePart(new_skin)
 			return TOPIC_REFRESH_UPDATE_PREVIEW
 
-		else if(has_flag(mob_species, HAS_SKIN_PRESET))
+	else if(href_list["skin_preset"])
+		if(!has_flag(mob_species, HAS_SKIN_PRESET))
+			return TOPIC_NOACTION
+		if(has_flag(mob_species, HAS_SKIN_PRESET))
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/general/03_body.dm:632]: this is a useless check now, it's the same as the one above

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**